packages/server-boilerplate/src/index.ts

```diff
@@ -20,4 +20,4 @@ export {
 } from './orchestrator';
 export * from './types';
 export * from './models';
-export { TestableServer } from './__tests__/utilities';
+export { TestableServer } from './testUtilities';
```

Comment:
Moved testUtilities out of `__tests__` because `__tests__` doesn't get ouput into dist, and other packages need these testUtilities.